src/cli-ux/prompt.ts

```diff
@@ -149,9 +149,9 @@ export function confirm(message: string): Promise<boolean> {
 /**
  * "press anykey to continue"
  * @param message - optional message to display to user
- * @returns Promise<void>
+ * @returns Promise<string>
  */
-export async function anykey(message?: string): Promise<void> {
+export async function anykey(message?: string): Promise<string> {
```

Comment:
My change has revealed that this code was broken. It was typed as returning void but actually it was returning a string.